### PR TITLE
UK : Change TVG-Country for some BBC channels

### PR DESCRIPTION
**Pull request by BellezaEmporium**
BBC Arabic = UK -> AR

BBC World News is INT.

### CODE REVIEW

File: channels/uk.m3u [hunk 1]
```diff
@@ -9,17 +9,17 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/akaal_tv/hls1_smart_akaa
 https://alaraby.cdn.octivid.com/alaraby/smil:alaraby.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ooiJwD5.png" group-title="",Al Hiwar (1080p)
 https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="BBC Arabic ARB" tvg-name="BBC Arabic ARB" tvg-country="UK" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oFGPG9Y.png" group-title="News",BBC Arabic (720p)
+#EXTINF:-1 tvg-id="BBC Arabic ARB" tvg-name="BBC Arabic ARB" tvg-country="UK;AR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oFGPG9Y.png" group-title="News",BBC Arabic (720p)
```
Comment (freearhey) [inline channels/uk.m3u:12]: "AR" code of Argentina. For Arab countries there is a code "MENA" (Middle East and North Africa). A list of other supported region codes can be found here: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md#supported-region-codes

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/uk.m3u (hunk 1 above)
Comment (themak1985) [reply channels/uk.m3u]: @freearhey MENA includes Iran and Turkey which BBC Arabic does not broadcast to, this is an example of a channel that would benefit from an Arab Speaking Countries region.

File: channels/uk.m3u [hunk 2]
```diff
@@ -9,7 +9,7 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/akaal_tv/hls1_smart_akaa
 https://alaraby.cdn.octivid.com/alaraby/smil:alaraby.stream.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="Arabic" tvg-logo="https://i.imgur.com/ooiJwD5.png" group-title="",Al Hiwar (1080p)
 https://mn-nl.mncdn.com/alhiwar_live/smil:alhiwar.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="BBC Arabic ARB" tvg-name="BBC Arabic ARB" tvg-country="UK;AR" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oFGPG9Y.png" group-title="News",BBC Arabic (720p)
+#EXTINF:-1 tvg-id="BBC Arabic ARB" tvg-name="BBC Arabic ARB" tvg-country="UK;MENA" tvg-language="Arabic" tvg-logo="https://i.imgur.com/oFGPG9Y.png" group-title="News",BBC Arabic (720p)
```
Comment (themak1985) [inline channels/uk.m3u:12]: @freearhey MENA includes Iran and Turkey which BBC Arabic does not broadcast to, this is an example of a channel that would benefit from an Arab Speaking Countries region.

Comment (freearhey) [reply channels/uk.m3u:12]: @themak1985 I just added an `ARAB` code for the [Arab world](https://en.wikipedia.org/wiki/Arab_world). I hope this helps.

**Review by freearhey — APPROVED** [review]

File: channels/uk.m3u (hunk 2 above)
Comment (bermufine) [reply channels/uk.m3u]: Can you update also for congo democratic republic all channels?
Please guys

**Pull request merged**